libr/bin/bin.c

```diff
@@ -417,6 +417,9 @@ R_IPI RBinXtrPlugin *r_bin_get_xtrplugin_by_name(RBin *bin, const char *name) {
 }
 
 static void r_bin_plugin_free(RBinPlugin *p) {
+	if (p && p->fini) {
+		p->fini ();
```

Comment:
Fini should take bin and binplugin too

Reply:
Maybe the binplugin is implicit

Reply:
problem is that RBinPluginFree is not taking RBin* so it doesnt have any context..

Reply:
uhm, the information that fini should free/finitialize is not actually stored in the plugin struct